pkg/daemon/config.go

```diff
@@ -24,12 +24,14 @@ type LinuxPTPConfUpdate struct {
 }
 
 type ptp4lConfSection struct {
-	options map[string]string
+	sectionName string
+	options     map[string]string
 }
 
 type ptp4lConf struct {
-	sections map[string]ptp4lConfSection
-	mapping  []string
+	sections     []ptp4lConfSection
+	mapping      []string
+	profile_name string
```

Comment:
since this has some changes that was fixed in 4.11 to add profile name.
Can you bring in  other changes too ?
https://github.com/openshift/linuxptp-daemon/pull/77/files

Reply:
it will help if we forklift  latest cloud-event-proxy top 4.10